drivers/ir_nec/Makefile.dep

```diff
@@ -0,0 +1,3 @@
+USEMODULE += isrpipe
+USEMODULE += periph_gpio_irq
+USEMODULE += ztimer_usec
```

Comment:
instead of using the periph_gpio_irq it needs to be required (this enables murdock to check if the feature is availible)

```suggestion
FEATURES_REQUIRED += periph_gpio
FEATURES_REQUIRED += periph_gpio_irq

USEMODULE += isrpipe
USEMODULE += ztimer_usec
```